src/CommunityToolkit.Maui.MediaElement/Extensions/SubtitleExtensions.windows.cs

```diff
@@ -0,0 +1,187 @@
+﻿using CommunityToolkit.Maui.Core;
+using CommunityToolkit.Maui.Core.Views;
+using CommunityToolkit.Maui.Primitives;
+using Microsoft.UI.Xaml.Media;
+
+namespace CommunityToolkit.Maui.Extensions;
+
+partial class SubtitleExtensions : Grid, IDisposable
+{
+	bool disposedValue;
+	bool isFullScreen = false;
+
+	static readonly HttpClient httpClient = new();
+	readonly Microsoft.UI.Xaml.Controls.TextBlock subtitleTextBlock;
+
+	List<SubtitleCue> cues;
+	IMediaElement? mediaElement;
+	MauiMediaElement? mauiMediaElement;
+	System.Timers.Timer? timer;
```

Comment:
most of these members are equals on other implementations, can we move it to a shared class?

Reply:
Created a new base shared base class